From 37986abc7e79673ece840a5975dd8531a1937e39 Mon Sep 17 00:00:00 2001
From: Dan Prince <dprince@redhat.com>
Date: Wed, 26 Feb 2014 12:57:07 -0500
Subject: [PATCH] Stop using the os-svc-install -n -c options.

We are repeatedly using the -n and -c options which don't
do anything. This removes the use of the unused options.

Partial-bug: #1285283

Change-Id: Ie67f3dfc1461ad41dcf130122902d4f1325e38ba
---
 elements/ceilometer/install.d/68-ceilometer                     | 2 +-
 elements/cinder/install.d/cinder-source-install/72-cinder       | 2 +-
 elements/glance/install.d/glance-source-install/75-glance       | 2 +-
 elements/heat/install.d/heat-source-install/05-heat             | 2 +-
 elements/horizon/install.d/horizon-source-install/100-horizon   | 2 +-
 elements/ironic/install.d/68-ironic                             | 2 +-
 elements/keystone/install.d/keystone-source-install/05-keystone | 2 +-
 elements/keystone/install.d/keystone-source-install/70-keystone | 2 +-
 elements/neutron/install.d/neutron-source-install/76-neutron    | 2 +-
 elements/nova/install.d/nova-source-install/74-nova             | 2 +-
 elements/swift/install.d/swift-source-install/75-swift          | 2 +-
 elements/tempest/install.d/51-tempest                           | 2 +-
 elements/tuskar/install.d/100-tuskar-api                        | 2 +-
 13 files changed, 13 insertions(+), 13 deletions(-)

diff --git a/elements/ceilometer/install.d/68-ceilometer b/elements/ceilometer/install.d/68-ceilometer
index 513b74f..3a23884 100755
--- a/elements/ceilometer/install.d/68-ceilometer
+++ b/elements/ceilometer/install.d/68-ceilometer
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eux
 
-os-svc-install -n ceilometer -u ceilometer -r /opt/stack/ceilometer
+os-svc-install -u ceilometer -r /opt/stack/ceilometer
 
 
 install -m 640 -o ceilometer -g ceilometer /opt/stack/ceilometer/etc/ceilometer/pipeline.yaml /etc/ceilometer
diff --git a/elements/cinder/install.d/cinder-source-install/72-cinder b/elements/cinder/install.d/cinder-source-install/72-cinder
index 9530364..99bda4b 100755
--- a/elements/cinder/install.d/cinder-source-install/72-cinder
+++ b/elements/cinder/install.d/cinder-source-install/72-cinder
@@ -3,7 +3,7 @@ set -eux
 
 install-packages lvm2 libssl-dev tgt
 echo "qpid-python" | tee -a /opt/stack/cinder/requirements.txt
-os-svc-install -n cinder -u cinder -r /opt/stack/cinder -c cinder-all
+os-svc-install -u cinder -r /opt/stack/cinder
 
 cp -a /opt/stack/cinder/etc/cinder/rootwrap* /etc/cinder
 cp /opt/stack/cinder/etc/cinder/api-paste.ini /etc/cinder/api-paste.ini
diff --git a/elements/glance/install.d/glance-source-install/75-glance b/elements/glance/install.d/glance-source-install/75-glance
index 4de1e6b..af79803 100755
--- a/elements/glance/install.d/glance-source-install/75-glance
+++ b/elements/glance/install.d/glance-source-install/75-glance
@@ -3,7 +3,7 @@ set -eux
 
 install-packages python-mysqldb libssl-dev
 
-os-svc-install -n glance -u glance -r /opt/stack/glance
+os-svc-install -u glance -r /opt/stack/glance
 
 ln -s /opt/stack/venvs/glance/bin/glance-manage /usr/local/bin/glance-manage
 
diff --git a/elements/heat/install.d/heat-source-install/05-heat b/elements/heat/install.d/heat-source-install/05-heat
index 23fe287..544c9f9 100755
--- a/elements/heat/install.d/heat-source-install/05-heat
+++ b/elements/heat/install.d/heat-source-install/05-heat
@@ -4,7 +4,7 @@ install-packages python-setuptools python-eventlet python-greenlet \
     python-httplib2 python-iso8601 python-lxml python-sqlalchemy \
     python-paste python-routes python-webob python-yaml
 
-os-svc-install -n heat -u heat -r /opt/stack/heat
+os-svc-install -u heat -r /opt/stack/heat
 
 cp /opt/stack/heat/etc/heat/policy.json /etc/heat/policy.json
 
diff --git a/elements/horizon/install.d/horizon-source-install/100-horizon b/elements/horizon/install.d/horizon-source-install/100-horizon
index 072bd7e..363f1eb 100755
--- a/elements/horizon/install.d/horizon-source-install/100-horizon
+++ b/elements/horizon/install.d/horizon-source-install/100-horizon
@@ -14,7 +14,7 @@ if [ -d /etc/apache2 ]; then
     rm -f /etc/apache2/sites-enabled/{000-default.conf,default.conf,default-ssl.conf}
 fi
 
-os-svc-install -n horizon -u horizon -r /opt/stack/horizon
+os-svc-install -u horizon -r /opt/stack/horizon
 
 # Django won't start otherwise:
 touch /etc/horizon/.secret_key_store
diff --git a/elements/ironic/install.d/68-ironic b/elements/ironic/install.d/68-ironic
index 7fecf0f..4c7741f 100755
--- a/elements/ironic/install.d/68-ironic
+++ b/elements/ironic/install.d/68-ironic
@@ -4,6 +4,6 @@ set -eux
 # need the libssl header files
 install-packages libssl-dev
 
-os-svc-install -n ironic -u ironic -r /opt/stack/ironic
+os-svc-install -u ironic -r /opt/stack/ironic
 
 install -d -m 0750 -o ironic -g ironic /etc/ironic
diff --git a/elements/keystone/install.d/keystone-source-install/05-keystone b/elements/keystone/install.d/keystone-source-install/05-keystone
index 4114a37..f84024e 100755
--- a/elements/keystone/install.d/keystone-source-install/05-keystone
+++ b/elements/keystone/install.d/keystone-source-install/05-keystone
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -eux
-os-svc-install -n keystone -u keystone -r /opt/stack/keystone -c "/opt/stack/keystone/bin/keystone-all"
+os-svc-install -u keystone -r /opt/stack/keystone
 
 # Workaround for https://bugs.launchpad.net/tripleo/+bug/1288425
 /opt/stack/venvs/keystone/bin/pip install -U kombu
diff --git a/elements/keystone/install.d/keystone-source-install/70-keystone b/elements/keystone/install.d/keystone-source-install/70-keystone
index 00e0fb0..810ad4c 100755
--- a/elements/keystone/install.d/keystone-source-install/70-keystone
+++ b/elements/keystone/install.d/keystone-source-install/70-keystone
@@ -3,7 +3,7 @@ set -eux
 
 DISTRO=`lsb_release -si` || true
 
-os-svc-install -n keystone -u keystone -r /opt/stack/keystone
+os-svc-install -u keystone -r /opt/stack/keystone
 os-svc-daemon keystone keystone keystone-all "--config-dir /etc/keystone"
 install -m 0755 -o keystone -g keystone -d /etc/keystone/ssl
 
diff --git a/elements/neutron/install.d/neutron-source-install/76-neutron b/elements/neutron/install.d/neutron-source-install/76-neutron
index bf41213..cb1eca0 100755
--- a/elements/neutron/install.d/neutron-source-install/76-neutron
+++ b/elements/neutron/install.d/neutron-source-install/76-neutron
@@ -8,7 +8,7 @@ install-packages openvswitch-switch openvswitch-datapath-dkms
 # for BUG: https://bugs.launchpad.net/neutron/+bug/1225232
 echo "qpid-python" | tee -a /opt/stack/neutron/requirements.txt
 
-os-svc-install -n neutron -u neutron -r /opt/stack/neutron
+os-svc-install -u neutron -r /opt/stack/neutron
 
 ln -sf /opt/stack/venvs/neutron/bin/neutron-rootwrap /usr/bin/neutron-rootwrap
 ln -sf /opt/stack/venvs/neutron/etc/neutron/rootwrap.d /etc/neutron/rootwrap.d
diff --git a/elements/nova/install.d/nova-source-install/74-nova b/elements/nova/install.d/nova-source-install/74-nova
index 45a13f8..f0fffe6 100755
--- a/elements/nova/install.d/nova-source-install/74-nova
+++ b/elements/nova/install.d/nova-source-install/74-nova
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -eux
 
-os-svc-install -n nova -u nova -r /opt/stack/nova
+os-svc-install -u nova -r /opt/stack/nova
 
 cp /opt/stack/nova/etc/nova/policy.json /etc/nova/policy.json
 cp /opt/stack/nova/etc/nova/api-paste.ini /etc/nova/api-paste.ini
diff --git a/elements/swift/install.d/swift-source-install/75-swift b/elements/swift/install.d/swift-source-install/75-swift
index 6273a21..9672e14 100755
--- a/elements/swift/install.d/swift-source-install/75-swift
+++ b/elements/swift/install.d/swift-source-install/75-swift
@@ -4,6 +4,6 @@ set -eux
 
 install-packages libffi-dev
 
-os-svc-install -n swift -u swift -r /opt/stack/swift
+os-svc-install -u swift -r /opt/stack/swift
 
 install -m 0755 -o root -g root /opt/stack/venvs/swift/bin/swift-ring-builder /usr/local/bin
diff --git a/elements/tempest/install.d/51-tempest b/elements/tempest/install.d/51-tempest
index 48e4732..0bc1632 100755
--- a/elements/tempest/install.d/51-tempest
+++ b/elements/tempest/install.d/51-tempest
@@ -4,7 +4,7 @@ set -ex
 
 install-packages augeas-tools
 
-os-svc-install -n tempest -u tempest -r /opt/stack/tempest
+os-svc-install -u tempest -r /opt/stack/tempest
 
 source /opt/stack/venvs/tempest/bin/activate
 cd /opt/stack/tempest
diff --git a/elements/tuskar/install.d/100-tuskar-api b/elements/tuskar/install.d/100-tuskar-api
index fb46776..329f0be 100755
--- a/elements/tuskar/install.d/100-tuskar-api
+++ b/elements/tuskar/install.d/100-tuskar-api
@@ -2,7 +2,7 @@
 set -eux
 
 install-packages swig
-os-svc-install -n tuskar -u tuskar -r /opt/stack/tuskar
+os-svc-install -u tuskar -r /opt/stack/tuskar
 
 install -d -m 0750 -o tuskar -g tuskar /etc/tuskar
 cp -a /opt/stack/tuskar/etc/tuskar/policy.json /etc/tuskar
-- 
1.8.5.3

